plugins/aggregators/quantile/README.md

```diff
@@ -0,0 +1,135 @@
+# Quantile Aggregator Plugin
+
+The quantile aggregator plugin aggregates specified quantiles for each numeric field
+per metric it sees and emits the quantiles every `period`.
+
+### Configuration
+
+```toml
+[[aggregators.quantile]]
+  ## General Aggregator Arguments:
+  ## The period on which to flush & clear the aggregator.
+  period = "30s"
+
+  ## If true, the original metric will be dropped by the
+  ## aggregator and will not get sent to the output plugins.
+  drop_original = false
+
+  ## Quantiles to output in the range [0,1]
+  # quantiles = [0.25, 0.5, 0.75]
+
+  ## Type of approximation
+  ## Supported are:
+  ##  "t-digest" -- approximation using centroids, can cope with large number of samples
+  ##  "exact R7" -- exact approximation also used by Excel or NumPy (Hyndman & Fan 1996 R7)
+  ##  "exact R8" -- exact approximation (Hyndman & Fan 1996 R8)
+  ## NOTE: Do not use "exact" approximations with large number of samples
+  ##       to not impair performance or memory consumption!
+  # approximator = "t-digest"
```

Comment:
It doesn't make sense to me to call this setting "approximator" when two of the options are exact, not approximate.  What do you think about calling it algorithm instead?

Reply:
Will do.